test/powershell/engine/Api/MultithreadedEngine.Tests.ps1

```diff
@@ -0,0 +1,198 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+Describe 'Multithreaded engine APIs' -Tags 'CI' {
```

Comment:
Can we change the name to be `"Task-based PowerShell async APIs"`? And the file name too, maybe `"TaskBasedAsyncPowerShellAPI.Tests.ps1"`.

Reply:
I like those better. Done.